File: beecrowd/beginner/c/1131.c
#include <stdio.h>

int main() {
    int inter_goals, gremio_goals, inter_victories, gremio_victories, count_grenais, draws, x = 1;

    inter_goals = gremio_goals = inter_victories = gremio_victories = count_grenais = draws = 0;

    do {
        scanf("%d %d", &inter_goals, &gremio_goals);

        count_grenais++;

        if (inter_goals > gremio_goals){ 
            inter_victories++;
        } else if (gremio_goals > inter_goals) {
            gremio_victories++;
        } else {
            draws++;
        }

        printf("Novo grenal (1-sim 2-nao)\n");
        scanf("%d", &x);
        
    } while (x == 1);

    printf("%d grenais\n", count_grenais);
    printf("Inter:%d\n", inter_victories);
    printf("Gremio:%d\n", gremio_victories);
    printf("Empates:%d\n", draws);
    
    if (inter_victories == gremio_victories){
        printf("Não houve vencedor\n");
    } else{
        printf("%s venceu mais\n", inter_victories > gremio_goals ? "Inter" : "Gremio");
    }

    return 0;
}

File: beecrowd/beginner/python/1131.py
gremio_victories = inter_victories = count_matches = draws = flag = 0

while 1:
    inter, gremio = map(int, input().split())

    count_matches += 1

    if gremio > inter:
        gremio_victories += 1
    elif inter > gremio:
        inter_victories += 1
    else:
        draws += 1

    while 1:
        print("Novo grenal (1-sim 2-nao)")

        op = int(input())

        if op == 2:
            flag = 1
            break
        elif op == 1:
            break
    
    if flag:
        break

print(f"{count_matches} grenais")
print(f"Inter:{inter_victories}")
print(f"Gremio:{gremio_victories}")
print(f"Empates:{draws}")

if inter_victories == gremio_victories:
    print("Não houve vencedor")
else:
    print(
        "Inter venceu mais"
        if inter_victories > gremio_victories
        else "Gremio venceu mais"
    )
